exampleQuiz.json

```diff
@@ -1,50 +0,0 @@
-{
-    "_id": "1923",
-    "name": "State Capitals",
-    "description": "The capitals of all 50 states in the U.S.",
-    "creation_date": "1559404138",
-    "tags": [
```

Comment:
Why did you delete the example quiz?

Reply:
Don't even know how that happened, fixed now